### PR TITLE
NFT table and NFT detail on v5

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/profile/\[profileAddress\]**](https://thirdweb-m39uvxd6k.thirdweb-preview.com/profile/[profileAddress]) |
|:-:|
|<a href="https://thirdweb-m39uvxd6k.thirdweb-preview.com/profile/[profileAddress]">Screenshot of /profile/[profileAddress]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/R5vuCG-nSscyB7DBDy06zMxagfsw9WX-Cqq3dOv15FBnBzTs-Unbmr1MCTYPU_cTug5ogPCyAn3Q6aXjYxZLig.png">(view full size)</a> |







Commit 6c5d3ef25241e667406b7bd495c5e2db9f0c2b2a (https://thirdweb-m39uvxd6k.thirdweb-preview.com).

File: contract-ui/hooks/useRouteConfig.tsx [hunk 1]
```diff
@@ -166,10 +165,12 @@ export function useContractRouteConfig(
...
-      isEnabled: extensionDetectedState({
-        contractQuery,
-        feature: ["ERC1155", "ERC721"],
-      }),
+      isEnabled: contract
+        ? extensionDetectedState({
+            contractQuery,
+            feature: ["ERC1155", "ERC721"],
+          })
+        : "disabled",
```
Comment (jnsdls) [inline contract-ui/hooks/useRouteConfig.tsx:173]: wouldn't this be `"loading"` ?

File: contract-ui/tabs/overview/page.tsx [hunk 2]
```diff
@@ -15,19 +15,21 @@ import { PublishedBy } from "components/contract-components/shared/published-by"
...
-  CustomContractOverviewPageProps
-> = ({ contractAddress }) => {
+export const ContractOverviewPage: React.FC<ContractOverviewPageProps> = ({
+  contractAddress,
+}) => {
   const { contract } = useContract(contractAddress);
   const contractTypeQuery = contractType.useQuery(contractAddress);
   const contractTypeData = contractTypeQuery?.data || "custom";
 
+  console.log({ contractAddress });
```
Comment (jnsdls) [inline contract-ui/tabs/overview/page.tsx:31]:
```suggestion
```

**Review by jnsdls — COMMENTED** [review]
mostly looks good

File: contract-ui/hooks/useRouteConfig.tsx (hunk 1 above)
Comment (nachoiacovino) [reply contract-ui/hooks/useRouteConfig.tsx]: reverting this change since we ended up doing this logic on the component render instead

**Pull request merged**